tests/test_data/quitoque.fr/quitoque.json

```diff
@@ -0,0 +1,82 @@
+{
+  "author": "QuiToque",
+  "canonical_url": "https://www.quitoque.fr/products/cuisse-de-canard-au-romarin-et-pommes-de-terre-sarladaises-13720",
+  "category": "Tradition",
+  "description": "Les pommes de terre sarladaises sont des patates sautées avec le gras du canard et parfumées à l'ail et au persil.",
+  "equipment": [
+    "four",
+    "sauteuse",
+    "cocotte"
+  ],
+  "host": "quitoque.fr",
+  "image": "https://image-repository-07c0d87b07ca0475fe83bb98.quitoque.fr/sylius_shop_product_cover/e3/72/16c5182dc42132ed776f094e98b0.webp",
+  "ingredient_groups": [
```

Comment:
@jknndy @strangetom I think I'll prepare a change soon so that `ingredient_groups` has to be omitted from the JSON if it isn't implemented on the scraper.  Currently it results in a lot of duplication in our test data.

(no need for you to worry about this @romainchassaigne - it's not related to your change, but is something about the library that I think we could improve)

Reply:
I've filed #1181 to track that suggestion, and will close/resolve this subthread.